skactiveml/classifier/_parzen_window_classifier.py

```diff
@@ -208,3 +237,10 @@ def predict_freq(self, X):
             for i in range(np.size(X, 0)):
                 F[i, :] = K[i, indices[i]] @ self.V_[indices[i], :]
         return F
+
+
+def _calculate_mean_gamma(N, variance, delta=(np.sqrt(2) * 1e-6)):
```

Comment:
@AlexanderBenz Please test this function independently from PWC. Use multiple values to check if it is correct.